Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { createClient } from '@supabase/supabase-js';\n\nconst supabaseUrl = 'https://kwunuyczgqvpnfsdueyl.supabase.co/';\nconst supabaseAnonKey =\n  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imt3dW51eWN6Z3F2cG5mc2R1ZXlsIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MzE5OTYzMjcsImV4cCI6MjA0NzU3MjMyN30.O9D9LB-vRuqEa-5Y4lFNAftCAXcm36g3nIUTb7ZOyhU';\n\nexport const supabase = createClient(supabaseUrl, supabaseAnonKey);
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
--- a/.env	(revision 8446b356a2554f61863874e65f04476cc7dddf7a)
+++ b/.env	(date 1738282856714)
@@ -1,7 +1,6 @@
-import { createClient } from '@supabase/supabase-js';
+EXPO_PUBLIC_API_KEY=6873eebcdfmshe6f2414415dce88p1346bfjsn16067c819f51
+EXPO_PUBLIC_API_HOST=restaurants-near-me-usa.p.rapidapi.com
+EXPO_PUBLIC_API_URL=https://restaurants-near-me-usa.p.rapidapi.com/
 
-const supabaseUrl = 'https://kwunuyczgqvpnfsdueyl.supabase.co/';
-const supabaseAnonKey =
-  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imt3dW51eWN6Z3F2cG5mc2R1ZXlsIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MzE5OTYzMjcsImV4cCI6MjA0NzU3MjMyN30.O9D9LB-vRuqEa-5Y4lFNAftCAXcm36g3nIUTb7ZOyhU';
-
-export const supabase = createClient(supabaseUrl, supabaseAnonKey);
\ No newline at end of file
+EXPO_PUBLIC_SUPABASE_URL=https://kwunuyczgqvpnfsdueyl.supabase.co/
+EXPO_PUBLIC_SUPABASE_ANON_KEY = eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imt3dW51eWN6Z3F2cG5mc2R1ZXlsIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MzE5OTYzMjcsImV4cCI6MjA0NzU3MjMyN30.O9D9LB-vRuqEa-5Y4lFNAftCAXcm36g3nIUTb7ZOyhU
\ No newline at end of file
